src/glyph/BBox.js

```diff
@@ -45,20 +45,24 @@ export default class BBox {
   }
 
   addPoint(x, y) {
-    if (x < this.minX) {
-      this.minX = x;
-    }
+    if (Math.abs(x) != Infinity) {
```

Comment:
ah. Might still be worth to change it to an `isFinite` check to bypass the `Math.abs` call entirely.